_posts/2022-02-09-hhvm-4.148.markdown

```diff
@@ -0,0 +1,22 @@
+---
+title: "HHVM 4.148"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.148 is released! This release marks the end of support of 4.140;
+4.143 - 4.146 remain supported, as do the 4.102 and 4.128 LTS releases.
+
+# Highlights
+
+- Fixed error where libelf may not be found
+- `CMAKE_BUILD_TYPE=Debug`
```

Comment:
Shall we remove this highlight, given that `CMAKE_BUILD_TYPE=Debug` is still broken due to other issues? See https://github.com/facebook/hhvm/issues/8984